manifests/compose/monitoring/prometheus/prometheus.yml

```diff
@@ -1,5 +1,5 @@
 global:
-  scrape_interval: 5s # Set the scrape interval to every 5 seconds. Default is every 1 minute.
+  scrape_interval: 3s # Set the scrape interval to every 5 seconds. Default is every 1 minute.
```

Comment:
check why changed scrape interval, and update comment accordingly

Reply:
setting scrape every 3 seconds rather than every 5 seconds, over a typical time window will collect significantly more data